integration/yarn/bin/alluxio-yarn.sh

```diff
@@ -60,12 +60,12 @@ tar -C $ALLUXIO_HOME -zcf $ALLUXIO_TARFILE \
   integration/yarn/bin/alluxio-worker-yarn.sh \
   integration/yarn/bin/alluxio-application-master.sh \
 
-JAR_LOCAL=${ALLUXIO_HOME}/assembly/server/target/alluxio-assembly-server-${VERSION}-jar-with-dependencies.jar
+JAR_LOCAL=${ALLUXIO_HOME}/assembly/integration/target/alluxio-assembly-integration-${VERSION}-jar-with-dependencies.jar
```

Comment:
this PR will focus on YARN change. @aaudiber  will have separate change for mesos